syntax = "proto3";

option java_multiple_files = true;

package laptopService.grpc;

service LaptopService{
    rpc buy(Empty) returns (Laptop);
    rpc sell(Laptop) returns (StringMessage);
}

message Empty{}

message Laptop{
    int32 serial_number = 1;
}

message StringMessage{
    string value = 1;
}
